include/zephyr/net/prometheus/pm_gauge.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2024 Mustafa Abdullah Kus, Sparse Technology
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file pm_gauge.h
+ * @brief https://prometheus.io/docs/concepts/metric_types/#gauge
+ */
+
+#ifndef ZEPHYR_INCLUDE_PM_GAUGE_H_
+#define ZEPHYR_INCLUDE_PM_GAUGE_H_
+
+#include <zephyr/sys/iterable_sections.h>
+#include <zephyr/net/prometheus/pm_metric.h>
+
+struct prometheus_gauge {
```

Comment:
Documentation header missing for the struct, also doxygen comments missing for the fields.